src/helm/benchmark/scenarios/contactcenter_convo_summarization_scenario.py

```diff
@@ -0,0 +1,122 @@
+"""
+Scenario for contact center conversational summarization.
+
+Loads input conversations defined in the ContactCenterConversationScenario
+and packs task specific input/output format for summarization tasks.
+
+Task structure
+
+    Conversation:
+            agent: message1
+            visitor: message2
+            ....
+    Summary:
+            summary of the conversation
+
+
+Example from the dataset
+    convo: Part of the Broad Road was closed to traffic on Sunday at about 18:00 GMT.
+                The three adults and three children have been taken to Altnagelvin Hospital
+                with non life-threatening injuries. The Fire Service, Northern Ireland Ambulance Service
```

Comment:
@chuanw24 should this be a more realistic example, and following the base schema we have for the input conversations – i.e. list of dict of messages info